doc/nrf/config_and_build/bootloaders_and_dfu/fw_update.rst

```diff
@@ -331,16 +353,58 @@ If this image has, in order of precedence, a *major*, *minor*, or *revision* val
    The optional label or build number specified after the ``+`` character is ignored when evaluating the version.
    An existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
 
-Hardware-based downgrade prevention
------------------------------------
+.. _ug_fw_update_downgrade_protection_hw:
+.. _bootloader_monotonic_counter:
+
+Hardware-based downgrade protection
+===================================
+
+.. bootloader_monotonic_counter_start
+
+You can implement hardware-based downgrade protection using a non-volatile monotonic counter.
+
+Counter updates are written to slots in the *Provision* area, with each new counter update occupying a new slot.
+For this reason, the number of counter updates, and therefore firmware version updates, is limited.
```

Comment:
As a reader, I find myself wondering what this "Provision area" is. Is it part of non-volatile memory? How can I find out where it is? Do we have documentation referring to this area somewhere else?
```suggestion
For this reason, the number of counter updates, and therefore firmware version updates, is limited. The *Provision* is a partition in non-volatile memory, and its location can be found using :ref:`Partition Manager <partition_manager_report>`.
```

Reply:
This is not "using the partition manager". However, a user would use the "parititon manager report" instead

Reply:
Fixed.